cli/tests/e2e/snapshots/test_ci/test_full_run/autofix-local/meta.json

```diff
@@ -2,7 +2,7 @@
   "meta": {
     "semgrep_version": "<sanitized version>",
     "repository": "checkout_project_name",
-    "repo_url": null,
+    "repo_url": "git@github.com:example/fake.git",
```

Comment:
is this ok? shouldn't this be an https:// url for links on the app to work?

Reply:
this is monkeypatched output to get determinism into the test. But I'm working on a better approach now.